Jropp/jason-ropp-js-iobp
# Polymer Upgrade: #3 Warning Message Component

## What It Does

This moves warning messages into a <warning-message> component that styles the text put into it.

- Make the code more readable.
- Reduces the amount of CSS in user-component, making it more maintainable.
- Makes it easier to keep consistency of warning messages across app if project expanded.

<warning-message> component has a slot that styles any <span> tags.

```html
<warning-message>
    <span>Message goes here</span>
</warning-message>
```

## How To Test

Improperly enter:
- email
- phone number

Warning messages should appear below the inputs.

Open new user form, then open an edit on an existing user. Message should display in new user form.

## Notes/Caveats

## Checklist

Under penalty of public shaming, I avow that I:


Browsers tested:


## Dependencies

- [x] #172 
- [x] #173
